src/locales/gb.json

```diff
@@ -59,14 +59,14 @@
+            "They were founded in France, the Annonay region, by ten diocesan priests who decided to radically live Community Life, following Jesus. Moved by the need of the time, they dedicated their efforts to teaching young people and preaching the Gospel.",
+            "Firm in their purpose, united by Common Life, they professed the vows of Chastity, Poverty and Obedience in 1822.",
+            "In the last 192 years they have been established in Canada, the United States, Mexico, Saint Lucia and Colombia.",
+            "In Colombia they have been present since 1987 in the city of Cali, Parroquia Nuestra Señora de la Asunción, in the Andrés Sanín neighborhood. Consistent with their choice, faithful to their charism for Education and Preaching, they lead the parish and the institute Nuestra Señora of the Assumption. ",
+            "With the firm purpose of responding thoroughly to the reality of Colombia, the Congregation of San Basilio made its second foundation in Colombia in 1995. The second house was established in the city of Medellín, initially in the Parish in Prado Center, later in the Divino Ecce Homo Parish in the Blanquizal neighborhood, currently and definitively in the San Basilio Parish, Olaya Herrera neighborhood, where a temple was built and a pastoral process led by Father Roberto Seguín was consolidated ",
+            "Aware of the importance of the education of future Basilians, in 1997 a house was established in the city of Bogotá. Although it was explored in the El Campín and July 20 neighborhoods with the Novitiate and the Scholasticate, in the year from 1999, we chose our own space called Basiliana Annonay House of Formation, in the Egypt neighborhood. ",
+            "Currently, in 2014, the Congregation of Basilian Priests is organizing the House for the Novitiate in the city of Santa Martha, as a budding experience, it will surely require immense efforts, but, redundant in the future of fruits for the work of the Kingdom here on Earth, among our People. "
         ],
-        "our-work": "Desde nuestra experiencia y carisma basiliano, realizar una nueva evangelización, que fomente la vida, en todos sus sentidos, promoviendo una Comunidad en Cristo, que alimente y celebre la liberación, organización, desarrollo y participación de la gente, para vivir aquí plenamente en hermandad, el Reino de Dios, ahora y por siempre."
+        "our-work": "From our experience and Basilian charism, carry out a new evangelization, that fosters life, in all its senses, promoting a Community in Christ, that nurtures and celebrates the liberation, organization, development and participation of people, to live here fully in brotherhood, the Kingdom of God, now and forever. "
```

Comment:
Try to avoid using some commas exactly as Spanish because in English gramma is different and sometimes not needed.